cheatsheets/XML_Security_Cheat_Sheet.md

```diff
@@ -511,14 +511,14 @@ public class parseDocument {
    for (int s = 0; s < nodeLst.getLength(); s++) {
      Node fstNode = nodeLst.item(s);
      if (fstNode.getNodeType() == Node.ELEMENT_NODE) {
-       Element fstElmnt = (Element) fstNode;
-       NodeList fstNmElmntLst = fstElmnt.getElementsByTagName("firstname");
-       Element fstNmElmnt = (Element) fstNmElmntLst.item(0);
-       NodeList fstNm = fstNmElmnt.getChildNodes();
+       Element fstElement = (Element) fstNode;
```

Comment:
This change (or any other) could of course be dropped. There's no reason to abbreviate the word `element` here. There really isn't a good reason to abbreviate `fst` and `lst` either (`Node` isn't abbreviated) -- to be honest, I can't really tell if those are `first` or `fast` / `list` or `last`.

I generally tolerate three letter sequences w/o inspecting unless they're obviously wrong. Here, I didn't notice either when reviewing the corpus.

Reply:
In the context you can tell when it's `first`/`last` (at the start, followed by `element` or `name`), or `list` (when it's at the end of an variable that's an array.

I could take or leave the change to make this more explicit, but I think it should be consistent. Either the original code, or fully expanding it (for example, this code still has `fstNmElementLst`, which looks out of place where you've expanded one part but not the other three.